persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageMapper.kt

```diff
@@ -365,6 +366,37 @@ object MessageMapper {
         lastEdit?.let { MessageEntity.EditStatus.Edited(it) }
             ?: MessageEntity.EditStatus.NotEdited
 
+    fun toEntityAssetMessageFromView(
+        id: String,
+        conversationId: QualifiedIDEntity,
+        contentType: MessageEntity.ContentType,
+        date: Instant,
+        visibility: MessageEntity.Visibility,
+        senderUserId: QualifiedIDEntity,
+        isEphemeral: Boolean,
+        senderName: String?,
+        selfUserId: QualifiedIDEntity?,
+        isSelfMessage: Boolean,
+        assetId: String?,
+        assetMimeType: String?,
+        assetHeight: Int?,
+        assetWidth: Int?,
+        assetDownloadStatus: MessageEntity.DownloadStatus?,
+        decodedAssetPath: String?
+    ): AssetMessageEntity {
+        return AssetMessageEntity(
```

Comment:
question: do we need to have another separate model for "AssetMessage", can't just `Message` with `AssetContent` be used? If it's due to performance then ok, but otherwise it can become confusing in the future - why and when to use `AssetMessage` or `Message` with `AssetContent` 🤔

Reply:
We need more fields than `Message` and `AssetContent` and `MessageDetailsView` is too big for this query